### PR TITLE
Update local-groups.mdx

### PR DESCRIPTION
**Pull request by Raxgahrax**
Add "Meshtastic France" and "Réseau Gaulix" to the France section in the local groups list.

### CODE REVIEW

File: docs/community/local-groups.mdx [hunk 1]
```diff
@@ -88,6 +88,11 @@ us on [Discord](https://discord.com/invite/ktMAKGBnBs) to add your group.
 - [Mesh Finland Discord](https://discord.com/invite/GHnaVAjqed)
 - [Mesh Finland Website](https://mesh-finland.github.io)
 
+## France
+
+- [Meshtastic France - Facebook](https://www.facebook.com/groups/meshtastic.france)
```
Comment (tekstrand) [inline docs/community/local-groups.mdx:93]: The name of this one will need to change to something like Mesh France, Meshtastic France Club, or something else that makes it clear this isn't an official Meshtastic entente.

We have some docs and guidelines around this here: https://meshtastic.org/docs/legal/licensing-and-trademark/#noncommercial-and-community-web-sites

**Review by tekstrand — CHANGES_REQUESTED** [review]
Some changes required for the France facebook group

File: docs/community/local-groups.mdx (hunk 1 above)
Comment (Raxgahrax) [reply docs/community/local-groups.mdx]: I made the change as requested.

**Review by Raxgahrax — COMMENTED** [review]
Change name group

File: docs/community/local-groups.mdx (hunk 1 above)
Comment (tekstrand) [reply docs/community/local-groups.mdx]: @Raxgahrax thanks for updating it here on the link. I am still seeing the name as the same on the actual page though.

Screenshot 2024-09-29 at 10 58 21 AM

Comment (Raxgahrax) [reply docs/community/local-groups.mdx]: Would it be possible to delete this pull request ? I only wanted to be helpful and I think I did things wrong

**Pull request closed without merging**